<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="editVector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vector</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editNumber">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number to count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="performButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Perform</string>
    </property>
   </widget>
   <widget class="QComboBox" name="compSwitch">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>=</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&gt;</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="sortingSwitch">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Low to High</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High to Low</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort comp.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="actionSwitch">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MAX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COUNT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SORT</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Operation</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultText">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
